src/browser/modules/EditorInfo/utils.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * returns the string representation
+ * of the requested node property value
+ * @param value
+ * value has neo4j data type
+ */
```

Comment:
@return {string} String representation of neo4j property value if valid data type, else empty string

Reply:
Added @return